plugin/trino-delta-lake/src/main/java/io/trino/plugin/deltalake/DeltaLakeMergeSink.java

```diff
@@ -191,9 +191,9 @@ private List<Slice> rewriteFile(Path sourcePath, FileDeletion deletion)
                     writerStats,
                     dataColumns);
 
-            DataFileInfo newFileInfo = rewriteParquetFile(sourcePath, deletion.rowsToDelete(), writer);
```

Comment:
There are a few typos in the commit description

Reply:
Fixed.